lib/shopify_api/resources/order.rb

```diff
@@ -7,7 +7,7 @@ def close;  load_attributes_from_response(post(:close, {}, only_id)); end
     def open;   load_attributes_from_response(post(:open, {}, only_id)); end
 
     def cancel(options = {})
-      load_attributes_from_response(post(:cancel, options, only_id))
+      load_attributes_from_response(post(:cancel, {}, options.to_json))
```

Comment:
Just so I understand correctly, looking for confirmation on this.. The endpoint returns an empty body due to: https://github.com/Shopify/shopify/blob/8ded28a6ed7a791ca5e87c3fcb54cce2316d079d/app/controllers/admin/orders_controller.rb#L335-L343